scripts/mods/mod_reforged/dynamic_spawns/parties/barbarian_parties.nut

```diff
@@ -9,9 +11,31 @@ local parties = [
 		UnitBlockDefs = [
 			{ BaseID = "UnitBlock.RF.BarbarianFrontline", RatioMin = 0.60, RatioMax = 1.00, DeterminesFigure = true },	// Vanilla: doesn't care about size
 			{ BaseID = "UnitBlock.RF.BarbarianSupport", RatioMin = 0.00, RatioMax = 0.07, ReqPartySize = 10 },			// Vanilla: Start spawning in armies of 15+; At 24+ a second drummer spawns
-			{ BaseID = "UnitBlock.RF.BarbarianDog", RatioMin = 0.00, RatioMax = 0.15, ReqPartySize = 5 },		// Vanilla: Start spawning in armies of 6+
-			{ BaseID = "UnitBlock.RF.BarbarianBeastmaster", RatioMin = 0.00, RatioMax = 0.10, ReqPartySize = 5 }		// Vanilla: Start spawning in armies of 7+ (singular case) but more like 9+
+			{ BaseID = "UnitBlock.RF.BarbarianDog", RatioMin = 0.00, RatioMax = 1.00, },		// Vanilla: Start spawning in armies of 6+
+			// want to add for dogs: if dogs in party >= 3 then reduce spawn chance by x (set RatioMax to 1.00 if we do this)
+			{ BaseID = "UnitBlock.RF.BarbarianBeastmaster", RatioMin = 0.00, RatioMax = 0.10, ReqPartySize = 5,  }		// Vanilla: Start spawning in armies of 7+ (singular case) but more like 9+
 		]
+
+		function generateIdealSize( _isLocation = false )
+		{
+			local startingResources = this.getSpawnProcess().getStartingResources();
+			if (startingResources >= 300)
```

Comment:
Considering how often you repeat the same three thresholds (300, 216, 164) we should define them globally somewhere so that we have them in one place. And submods can easier adjust them all at once

Something like `::Reforged.Const.DynamicSpawn.IdealSize.High`
@LordMidas